app/options.html

```diff
@@ -18,43 +18,43 @@
                 <div class="col-md-8 col-md-offset-2">
                     <div class="jumbotron">
                         <img src="images/icon-128.png" class="pull-right" id="icon128"/>
-                        <h1>FullyFeedly</h1>
-                        <p>Show the full content of the articles in Feedly.com</p>
+                        <h1 data-message="appName">FullyFeedly</h1>
+                        <p data-message="appDescription">Show the full content of the articles in Feedly.com</p>
```

Comment:
appName et appDescription key is used by the extention in the list of extensions.
You must put it in the English Translation.

For the "Show article preview", it's in javascript, and if you don't put it in the English translation, there will have no text in the button since the chrome.i18n.getMessage('showFullArticle') return nothing if there are no translation.
Or put, chrome.i18n.getMessage('showFullArticle') || 'Show Full Article' in the code.
But in this will, why use i18n for all errors messages ?

And the english file, must be the start translation file.